SSD/ssdDriver.cpp

```diff
@@ -1,14 +1,50 @@
 #include "device.h"
+#include <iostream>
+#include <string>
+#include <vector>
 
-class SSDDriver : public Device{
+using std::string;
+using std::vector;
+
+class SSDDriver : public Device {
 public:
+  void run(int argc, char *argv[]) {
+    bool result = false;
+    if (argc < 3)
+      std::exception();
+    if (argv[0] != "ssd.exe")
+      std::exception();
```

Comment:
질문 있습니다.
throw 없이 std::exception() 만 써도 프로그램 종료되나요...?

```suggestion
    if (argc < 3)
      throw std::exception();
    if (argv[0] != "ssd.exe")
      throw std::exception();
```